mne/utils/mixin.py

```diff
@@ -109,12 +109,18 @@ def __getitem__(self, item):
 
             3. ``epochs[['name_1', 'name_2', ... ]]``: Return ``Epochs`` object
                with a copy of the subset of epochs corresponding to multiple
-               experimental conditions as specified by
-               ``'name_1', 'name_2', ...`` .
-
-               If conditions are separated by '/', selects every item
-               containing every list tag (e.g. ['audio', 'left'] selects
-               'audio/left' and 'audio/center/left', but not 'audio/right').
+               experimental conditions. The list of names is treated as an
+               inclusive-or condition; if *none* of the provided names match
+               any epoch labels, a ``KeyError`` will be raised.
+
+               If epoch labels are tags separated by '/', then providing
+               multiple tags as *separate list entries* (e.g.,
+               ``epochs[['audio', 'left']]``) will also act as an inclusive-or
```

Comment:
isn't that repeating what is just said above?

Reply:
I think @drammock means to say that `[["left", "audio"]]` would work for both kinds of epochs:

1. those that use the `/` style: `"audio/left"`
1. those that have just single keywords: `"audio"` for one epoch, `"left"` for another epoch